test/integ/cli/test_cli_attachment.py

```diff
@@ -0,0 +1,285 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+Integ tests for the CLI attachment commands.
+"""
+
+import os
+import json
+from click.testing import CliRunner
+from dataclasses import asdict
+
+import pytest
+import tempfile
+
+from deadline_test_fixtures.job_attachment_manager import JobAttachmentManager
+
+from deadline.client.cli import main
+from deadline.client.cli._groups.attachment_group import cli_attachment
+from deadline.job_attachments.asset_manifests import (
+    HashAlgorithm,
+    hash_data,
+)
+from deadline.job_attachments.models import PathMappingRule
+
+TEST_FILE_SIZE = 14
+MOCK_MANIFEST_CASE = {
+    "TEST_CASE_1": {
+        "hashAlg": "xxh128",
+        "manifestVersion": "2023-03-03",
+        "paths": [
+            {
+                "hash": "19a71beb47d7cc2d654ac4637e680c88",
+                "mtime": 1720199667787520,
+                "path": "files/file2.txt",
+                "size": TEST_FILE_SIZE,
+            }
+        ],
+        "totalSize": TEST_FILE_SIZE,
+    },
+    "TEST_CASE_2": {
+        "hashAlg": "xxh128",
+        "manifestVersion": "2023-03-03",
+        "paths": [
+            {
+                "hash": "b03f20b08a76635964ab008a10cd20a8",
+                "mtime": 1720199667787520,
+                "path": "files/file1.txt",
+                "size": TEST_FILE_SIZE,
+            }
+        ],
+        "totalSize": TEST_FILE_SIZE,
+    },
+}
+
+
+class TestAttachment:
+
+    @pytest.fixture
+    def temp_dir(self):
```

Comment:
nit, the function is temp_dir, but it actually creates temp files. Maybe `temp_dir_and_files` ?

Reply:
Synced offline, since this returns dir, will keep it as-is.